### ISSUE TITLE
[Other - Accessibility] Single-click destructive action. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60037]
#### VFS team name
Benefits Team 2

#### VFS product name
Financial Status Report (FSR)

#### VFS feature name
FSR Enhancements

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
In the parts of the workflow where cards of items are added there is a delete button, which is a single-click destructive action. When clicking delete, the item is removed without any user confirmation. Additionally, there's nothing announced by screen readers to indicate that a change has occurred --- focus just moves to the next focusable element.

It's easy to imagine some scenarios where a user might accidentally trigger the delete button and not realize that they did it. Screen reader users may hit the wrong key but think they hit [Tab]. A low-vision user on a touch device might pinch-to-zoom and accidentally tap the delete button in the process, and not see the issue disappear.

Since users get an opportunity to review everything prior to submission, this is probably not a violation of experience standard 03.09 or WCAG 3.3.4. But I do think it's in the spirit of WCAG 3.3.4.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Two recommended changes:

1. Some kind of status message that receives focus when an issue has been removed, similar to the background-only alerts in use when changing contact information. If a visible status message isn't practical, at least some aria-live update should be announced to screen reader users.
2. Some kind of intermediate confirmation step between a user clicking Remove and the issue actually being removed. I believe the team working on the Supplemental Claims Form 20-0995 is exploring using a modal and intends to contribute back to the pattern in the design system. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by mgleo** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/56082 Similar ticket with modal implementation for Supplemental Claims

**Comment by HeatherRienks** [comment]
Addressing this in ticket #60303
cc: @Tom-Davis

**Comment by HeatherRienks** [comment]
#60303 is actually a duplicate ticket. Work is being addressed in #64425. Closing!
